pylzy/lzy/api/v1/remote/runtime.py

```diff
@@ -186,12 +189,12 @@ async def exec(
                 )
 
     async def abort(self) -> None:
-        client = self.__workflow_client
+        client = self.__lzy_client
         if not self.__running:
             return
         try:
             await client.abort_workflow(cast(LzyWorkflow, self.__workflow).name, self.__execution_id,
```

Comment:
Maybe it would be better to do one cast to a temp var workflow at start of this function?

Reply:
fixed